packages/synapse-react-client/src/mocks/chat/mockChat.ts

```diff
@@ -16,7 +16,7 @@ export const mockCreateAgentSessionRequest: CreateAgentSessionRequest = {
 }
 export const mockAgentSession: AgentSession = {
   sessionId: mockChatSessionId,
-  agentAccessLevel: AgentAccessLevel.WRITE_YOUR_PRIVATE_DATA,
+  agentAccessLevel: AgentAccessLevel.PUBLICLY_ACCESSIBLE,
```

Comment:
For now, when mocking only the PUBLICLY_ACCESSIBLE functions correctly (since the mock response always returns an agent with this hard-coded access level)